apps/web-staking/src/app/components/summary/SummaryComponent.tsx

```diff
@@ -112,6 +112,7 @@ const SummaryComponent = ({ isBannedPool, poolFromDb }: { isBannedPool: boolean,
   };
 
   const completeKeyRequest = async (requestIndex: number, numKeys: number) => {
+    setUnstakeRequestIndex(requestIndex);
```

Comment:
Can you give me some context for this update, are we sure the bug only happens whe you try to claim a key unstake request not an esXai one ?

Reply:
I traced the crash to that request index variable being null after a user claims a pool reward, but has not refreshed the page.

Setting the variable there resolved the crash and provided the expected behavior outlined in the ticket. The ticket doesn't mention esXai unstake, but if it should have just let me know what needs to be done!

Reply:
In the ticket it just says unstake request so we should asume it meant both. If you test and you can confirm that it only happens for key unstake Im happy and we can merge :)

Reply:
Will do, will report back after checking!

Reply:
This has been resolved. The error was also present on unstake esXai but has now been fixed!